snuba/consumers/consumer.py

```diff
@@ -113,6 +113,25 @@ def __reduce_ex__(
             )
 
 
+class LatencyRecorder:
+    def __init__(self) -> None:
+        self._sum = 0.0
+        self._max: Optional[float] = None
+
+    def record(self, latency: float) -> None:
```

Comment:
let's call it `latency_seconds` since we're assuming we have to multiply by 1000 to get ms later